Makefile

```diff
@@ -128,7 +128,8 @@ generate: controller-gen
 # Generate manifests e.g. CRD, RBAC etc.
 .PHONY: manifests
 manifests: controller-gen
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+#	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:maxDescLen=0 webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Comment:
Why `maxDescLen=0`?

Reply:
If you apply the CRD, you will get this error:
```console
The CustomResourceDefinition "[nificlusters.nifi.konpyutaika.com](http://nificlusters.nifi.konpyutaika.com/)" is invalid: metadata.annotations: Too long: must have at most 262144 bytes 
```

We add this to the skip description field : https://github.com/kubernetes-sigs/controller-tools/issues/441#issuecomment-650483628

Reply:
Ah, gotcha. I've seen this a lot in ArgoCD, which forces you to re-create the resource instead of applying. Makes sense!